pkg/image/decompress.go

```diff
@@ -2,14 +2,94 @@ package image
 
 import (
 	"archive/tar"
-	"fmt"
+	"compress/gzip"
 	"io"
-	"os"
 	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
 )
 
-func IsCompressed(srcFile string) bool {
-	fExt := filepath.Ext(srcFile)
+var SupportedCompressionExtensions = []string{
+	Gz,
+}
+
+var SupportedArchiveExtentions = []string{
```

Comment:
we should keep all of the extension in the same file. Currently we have some here and in _pkg/image/validate.go_